### PR TITLE
fix:  New App ID

### PR DESCRIPTION
**Pull request by cdnninja**

### CODE REVIEW

**Comment by cdnninja** [comment]
Just realized this is duplicate of #7  however leaving it since this approach is replacing the app id vs adding another.

**Comment by Hacksore** [comment]
Looks like only @neoPix can merge this, once this is done I can merge the bluelinky change.

https://github.com/Hacksore/bluelinky/pull/251/files

**Comment by cdnninja** [comment]
I have revised this and left the old Hyundai App ID in.  Since this one still works.  Old kia one doesn't work so not worth leaving.

**Review by neoPix — APPROVED** [review]

**Pull request merged**
